.travis.yml

```diff
@@ -18,19 +22,23 @@ jobs:
       script: python ./scripts/typecheck_tests.py
 
     - name: Lint using pre-commit
-      python: 3.8
+      python: 3.9-dev
```

Comment:
Why `-dev`?

Reply:
As far as I can see Travis CI still hasn't added support for stable release ([issue](https://travis-ci.community/t/python-3-9-dev-support/6703) and [versions list](https://docs.travis-ci.com/user/languages/python/#python-versions)).

Reply:
That list contains 3.9 now.

Reply:
@MarcinWieczorek please, send one more PR 🙂